server/template.yml

```diff
@@ -294,8 +320,8 @@ Outputs:
     Value: !Sub "https://${ServerlessRestApi}.execute-api.${AWS::Region}.amazonaws.com/prod/"
   UploadHandlerFunction:
     Value: !Ref UploadHandler
-  PostProcessHandlerFuntion:
-    Value: !Ref PostProcessHandler
+  JobCompletionStatusFuntion:
+    Value: !Ref JobCompletionStatus
   FileCompressApi:
     Description: "API Gateway endpoint URL for Prod stage for file compression function"
     Value: !Sub "https://${ServerlessRestApi}.execute-api.${AWS::Region}.amazonaws.com/Prod/fileCompress/"
```

Comment:
I'm not exactly sure why, but the send email one is missing which is in develop branch
```
 SendEmailFunction:
    Value: !Ref SendEmail
```

Reply:
It's present identically in both.